styles/go.dev/catppuccin.user.css

```diff
@@ -0,0 +1,930 @@
+    img.go-Footer-googleLogoImg, img.Footer-googleLogoImg {
+      @svg: escape('<svg width="74" height="24" xmlns="http://www.w3.org/2000/svg"><path d="M.128 9.276c0-5.105 4.32-9.261 9.457-9.261 2.842 0 4.865 1.107 6.388 2.552l-1.796 1.785c-1.091-1.017-2.569-1.807-4.592-1.807-3.75 0-6.683 3.004-6.683 6.731s2.932 6.732 6.683 6.732c2.432 0 3.82-.971 4.706-1.853.727-.722 1.204-1.761 1.386-3.184H9.585V8.44h8.57c.091.451.137.993.137 1.58 0 1.898-.523 4.248-2.206 5.92-1.636 1.693-3.728 2.597-6.5 2.597-5.139 0-9.458-4.156-9.458-9.262m25.09 6.914c-1.819 0-3.387-1.491-3.387-3.615 0-2.146 1.568-3.614 3.387-3.614s3.387 1.468 3.387 3.614c0 2.124-1.569 3.615-3.387 3.615m0-9.578c-3.32 0-6.024 2.507-6.024 5.963 0 3.434 2.705 5.964 6.024 5.964s6.024-2.53 6.024-5.964c0-3.456-2.706-5.963-6.024-5.963m13.142 9.578c-1.82 0-3.388-1.491-3.388-3.615 0-2.146 1.569-3.614 3.387-3.614s3.387 1.468 3.387 3.614c0 2.124-1.568 3.615-3.387 3.615m0-9.578c-3.319 0-6.024 2.507-6.024 5.963 0 3.434 2.705 5.964 6.024 5.964s6.024-2.53 6.024-5.964c0-3.456-2.705-5.963-6.024-5.963m13.107 9.578c-1.818 0-3.341-1.514-3.341-3.592 0-2.1 1.523-3.637 3.341-3.637 1.796 0 3.206 1.536 3.206 3.637 0 2.078-1.41 3.592-3.206 3.592m3.024-9.217v.972h-.091c-.591-.7-1.728-1.333-3.16-1.333-3 0-5.751 2.62-5.751 5.986 0 3.343 2.75 5.941 5.75 5.941 1.433 0 2.57-.633 3.16-1.356h.092v.859c0 2.281-1.228 3.501-3.206 3.501-1.614 0-2.614-1.152-3.023-2.123l-2.296.949c.66 1.581 2.41 3.524 5.32 3.524 3.091 0 5.705-1.808 5.705-6.212V6.972zm6.956 11.204h-2.637V.647h2.637zm7.024-9.261c1.046 0 1.932.52 2.228 1.265l-5.365 2.214c-.068-2.305 1.796-3.48 3.137-3.48m.205 7.275c-1.342 0-2.296-.61-2.91-1.807l8.025-3.298-.273-.678c-.5-1.333-2.024-3.795-5.138-3.795-3.092 0-5.66 2.417-5.66 5.963 0 3.344 2.546 5.964 5.956 5.964 2.75 0 4.342-1.672 5-2.643L71.63 14.54c-.682.994-1.614 1.649-2.955 1.649" fill="@{text}" fill-rule="evenodd"/></svg>');
+      content: url('data:image/svg+xml,@{svg}');
+    }
+    // Menu Active Selector
+    img.TabSection-underline {
+      @svg: escape('<svg width="51" height="4" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M0 4a4 4 0 0 1 4-4h43a4 4 0 0 1 4 4z" fill="@{accent-color}"/></svg>');
+      content: url('data:image/svg+xml,@{svg}');
+    }
+    // /doc/install icon
+    img.CopyPaste-icon {
+      @svg: escape('<svg width="14" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M10.992 0h-6L4 .992V4H.992L0 4.992V15.04l.992.96H9.04l.96-.96V12h3.12l.88-.96V2.992zm0 1.408 1.584 1.584h-1.584zm-2 13.584h-8v-10H4v6.048l.992.96h4zm4-4h-8v-10H10V4h2.992z" fill="@{text}"/></svg>');
+      content: url('data:image/svg+xml,@{svg}');
+    }
+    // Code Snippet Highlights
+    pre {
+      .comment {
+        color: @overlay2;
+      }
+      ins {
+        color: @blue;
+      }
+    }
+    // Companies in Main Page Contrast
+    & when (@lookup = latte) {
+      .WhoUsesCaseStudyList-caseStudyLink {
+        background-color: @surface0;
+      }
+    }
```

Comment:
Why only in latte?

Reply:
The contrast for the grid of company logos in latte in https://go.dev/ is not very good.

See without: 
![image](https://github.com/catppuccin/userstyles/assets/22081694/04f3d2e9-2547-4b2b-bb52-6e114c2466db)

With:
![image](https://github.com/catppuccin/userstyles/assets/22081694/92943183-e008-4cd2-ab2a-2b1e91073e7c)

Reply:
Ah, fair enough! Good choice.